_migrations/migration-phases/verifying-tools-for-migration.md

```diff
@@ -0,0 +1,196 @@
+---
+layout: default
+title: Verifying tools for migrations
+nav_order: 70
+parent: Migration phases
+---
+
+# Verifying tools for migration
+
+Before using the Migration Assistant, take the following steps to verify that you're cluster is ready for migration.
+
+## Snapshot creation verification
+
+Verify that a snapshot can be created from your cluster and used for metadata and backfill scenarios.
+
+### Installing the Elasticsearch S3 Repository plugin
+
+The snapshot needs to be stored in a location that the Migration Assistant can access. This guides uses AWS S3 as  and the Migration Assistant creates an S3 bucket for this purpose. Therefore, it is necessary to install the [Elasticsearch S3 Repository](https://www.elastic.co/guide/en/elasticsearch/plugins/7.10/repository-s3.html) plugin on your source nodes (https://www.elastic.co/guide/en/elasticsearch/plugins/7.10/repository-s3.html).
```

Comment:
```suggestion
The snapshot needs to be stored in a location that the Migration Assistant can access. This guides uses AWS S3 as and the Migration Assistant creates an S3 bucket for this purpose. Therefore, it is necessary to install the [Elasticsearch S3 Repository](https://www.elastic.co/guide/en/elasticsearch/plugins/7.10/repository-s3.html) plugin on your source nodes (https://www.elastic.co/guide/en/elasticsearch/plugins/7.10/repository-s3.html).
```

Reply:
@sumobrian: Wondering if there's a good way to make this more cloud agnostic. For example, adding a sentence such as "This guide uses AWS S3 as and the Migration Assistant creates an S3 bucket for this purpose, but the steps can be modified for your own cloud provider."